library/std/arithmetic.qs

```diff
@@ -289,4 +316,167 @@ namespace Microsoft.Quantum.Arithmetic {
         }
     }
 
+    //
+    //
+    //      New arithmetic operations starts here.
+    // Once it is done, previous implementation will be removed.
+    //
+    //
+
+    //
+    // Operation: Add      |    Ripple-carry    | Carry look-ahead
+    // ____________________________________________________________
+    // y += 5              |  RippleCarryIncByL✔|              N/A
+    // y += x              | RippleCarryIncByLE✔| LookAheadIncByLE
+    // z = x + 5 (z was 0) |                N/A |              N/A
+    // z = x + y (z was 0) |   RippleCarryAddLE✔|   LookAheadAddLE
+    // z += x + 5          |                N/A |              N/A
+    // z += x + y          |                N/A |              N/A
+    //
+
+    /// # Summary
+    /// Increment a little-endian ys register by a BigInt number c
+    ///
+    /// # Description
+    /// Compute ys += c modulo 2ⁿ, where ys is a little-endian register
+    /// of length n > 0, c >= 0 is a BigInt constant, and c < 2ⁿ.
+    /// This operation can be used when the choice of implementation
+    /// is not important.
```

Comment:
I think in most cases, the user will choose this operation. We could phrase this sentence more positively by writing something like "If the choice of implementation is of importance, use ... or ..." with references to the possible implementations.

Reply:
Updated text.